script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -212,32 +223,33 @@ I hear someone call out my name.
 [sha:5ee267336791ffa2da98f979e0ded3d9e26f226d]{
 -- Page 138, Offset 5206.
 -- ……顔をあげる。
-......I look up
+...I look up.
 }
 [sha:27e37a8c85b1206f00e65a7bfd8b7ce33c39a21f]{
 -- Page 138, Offset 5207.
 -- そこには、もう何十年かぶりに思える、先輩の姿があった。
- There, for what feels like the first time in decades, I see Senpai.
+Standing there in front of me is Senpai.
```

Comment:
revert

Reply:
I like ross' version better, sounds more natural to me.

Reply:
idk, old version reads more naturally to me and doesn't have the pause.